tests/sles4sap/sapconf.pm

```diff
@@ -84,6 +84,10 @@ sub run {
 
     $self->select_serial_terminal;
 
+    # Stop & disable tuned service to avoid conflict with active saptune
+    assert_script_run("systemctl stop tuned");
+    assert_script_run("systemctl disable tuned");
```

Comment:
It works, but FYI these 2 lines can be replaced by `systemctl disable --now tuned` for SLE15+, not sure for SLE12.
But that's fine :-)

Reply:
You can also use `systemctl()` from `Utils::Systemd` instead of `assert_script_run("systemctl ...")`

Reply:
The `--now` option doesn't work on older versions of Systemd.